src/workloads/query/TimeSeriesLastpointHighCardinality.yml

```diff
@@ -0,0 +1,337 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
```

Comment:
In the form you mention that latencies for MetaAscTimeAsc.RunLastPointQuery*, MetaAscTimeDesc.RunLastPointQuery*, MetaDescTimeDesc.RunLastPointQuery*, MetaDescTimeAsc.RunLastPointQuery* are the key metrics we care about here. Can you add a line in the description saying this.

(Also in the form you call out AverageLatency. I'm pretty sure for Genny workloads we only really track Latency50thPercentile and Latency95thPercentile)